### PR TITLE
nonempty-array dependency branch was merged and deleted

### PR DESCRIPTION
**Pull request by telser**
nonempty-array dependency needs to change branch

### CODE REVIEW

**Comment by jdegoes** [comment]
Review by @garyb

File: bower.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "tests"
   ],
   "dependencies": {
-    "purescript-nonempty-arrays": "garyb/purescript-nonempty-array#0.7-updates",
+    "purescript-nonempty-arrays": "stevejb71/purescript-nonempty-array#master",
```
Comment (garyb) [inline bower.json]: Can we use `"purescript-nonempty-arrays": "68d6e04cd1874f01489a1de4b7dca5c7186f88d0"` instead please? That's the hash for the merged branch.

**Comment by telser** [comment]
I made that change @garyb

**Comment by garyb** [comment]
Thanks! Could you squash the commits too please?

**Comment by garyb** [comment]
Actually, I think I'm going to switch this over to use the generic [`purescript-nonempty`](https://github.com/paf31/purescript-nonempty) instead, just waiting for a PR to be merged over there first. Thanks though!

**Pull request closed without merging**